config/locales/en.yml

```diff
@@ -824,6 +826,9 @@ Make sure these answers are correct."
     offered: "Times previously offered since becoming available"
 
   warnings:
+    organisation:
+      deactivate: "All schemes and users at this organisation will be deactivated. All the organisation's relationships will be removed. It will no longer be possible to create logs for this organisation."
+      reactivate: "All schemes, users, and relationships that were active when this organisation was deactivated will be reactivated."
```

Comment:
This is copied from the designs - do we usually use the Oxford comma in situations where it's not necessary?